core/templates/pages/exploration-player-page/exploration-player-page.module.ts

```diff
@@ -38,16 +38,16 @@ import { InteractionExtensionsModule } from 'interactions/interactions.module';
 import { LearnerLocalNavComponent } from './layout-directives/learner-local-nav.component';
 import { FlagExplorationModalComponent } from './modals/flag-exploration-modal.component';
 import { FeedbackPopupComponent } from './layout-directives/feedback-popup.component';
-import { MatButtonModule } from '@angular/material/button';
 import { ExplorationSuccessfullyFlaggedModalComponent } from './modals/exploration-successfully-flagged-modal.component';
+import { MaterialModule } from 'components/material.module';
 
 @NgModule({
   imports: [
     BrowserModule,
     HttpClientModule,
     InteractionExtensionsModule,
     NgbModalModule,
-    MatButtonModule,
+    MaterialModule,
```

Comment:
MaterialModule is a part of exports in the SharedComponents module. Why are we importing it here again?

Reply:
fixed.